### PR TITLE
Update unitary calculators.

### PR DESCRIPTION
**Pull request by sergeisakov**
Replace generated unitary calculators with handwritten unitary calculators.

### CODE REVIEW

File: tests/make.sh [hunk 1]
```diff
@@ -44,8 +44,9 @@ g++ -O3 -I$path_to_include -L$path_to_lib -mavx2 -mfma -fopenmp -o statespace_av
 g++ -O3 -I$path_to_include -L$path_to_lib -mavx512f -mbmi2 -fopenmp -o statespace_avx512_test.x statespace_avx512_test.cc -lgtest -lpthread
 g++ -O3 -I$path_to_include -L$path_to_lib -fopenmp -o statespace_basic_test.x statespace_basic_test.cc -lgtest -lpthread
 g++ -O3 -I$path_to_include -L$path_to_lib -msse4 -fopenmp -o statespace_sse_test.x statespace_sse_test.cc -lgtest -lpthread
-g++ -O3 -I$path_to_include -L$path_to_lib -mavx2 -mfma -fopenmp -o unitary_calculator_avx_test.x unitary_calculator_avx_test.cc -lgtest -lpthread
-g++ -O3 -I$path_to_include -L$path_to_lib -mavx512f -mfma -fopenmp -o unitary_calculator_avx512_test.x unitary_calculator_avx512_test.cc -lgtest -lpthread
+g++ -O3 -I$path_to_include -L$path_to_lib -mavx2 -mfma -fopenmp -o unitary_calculator_avx_test.x unitary_calculator_avx_nobmi2_test.cc -lgtest -lpthread
```
Comment (95-martin-orion) [inline tests/make.sh:47]: Should the `*_nobmi2_test` suffix be on the `.x` file instead? There isn't a `.cc` file with this name.

**Review by 95-martin-orion — CHANGES_REQUESTED** [review]
One issue in the `make` script, otherwise this looks in line with the previous simulator changes.

File: tests/make.sh (hunk 1 above)
Comment (sergeisakov) [reply tests/make.sh]: Fixed. Thank you for catching this.

File: lib/unitary_calculator_avx.h [hunk 2]
```diff
@@ -17,21 +17,22 @@
...
-#include "bits.h"
+#include "simulator.h"
 #include "unitaryspace_avx.h"
 
 namespace qsim {
 namespace unitary {
 
 /**
- * Quantum circuit unitary calculator with AVX vectorization.
+ * Quantum circuit simulator with AVX vectorization.
```
Comment (MichaelBroughton) [inline lib/unitary_calculator_avx.h:32]: Nit: word change here and other unitary_calulcator_*** variants.

**Review by 95-martin-orion — APPROVED** [review]

**Review by MichaelBroughton — APPROVED** [review]
LGTM

File: lib/unitary_calculator_avx.h (hunk 2 above)
Comment (sergeisakov) [reply lib/unitary_calculator_avx.h]: Fixed.

**Pull request merged**